File: GamesaveCloudLib/SaveGameFile.vb
Public Class SaveGameFile
    Public savegame As SaveGame
    Public path As String
    Public oldPath As String
    Public changeType As String

    Public Sub New(savegame As SaveGame, path As String, oldPath As String, changeType As String)
        Me.savegame = savegame
        Me.path = path
        Me.oldPath = oldPath
        Me.changeType = changeType
    End Sub

End Class

File: GamesaveCloudLib/SaveGameFile.cs

namespace GamesaveCloudLib
{
    public class SaveGameFile
    {
        public SaveGame savegame;
        public string path;
        public string oldPath;
        public string changeType;

        public SaveGameFile(SaveGame savegame, string path, string oldPath, string changeType)
        {
            this.savegame = savegame;
            this.path = path;
            this.oldPath = oldPath;
            this.changeType = changeType;
        }

    }
}